geometry/geometry_config.py

```diff
@@ -789,11 +789,11 @@
     c.NuTauTarget.WallZDim = c.NuTauTarget.BrZ
 
     if (c.NuTauTT.design==4):
-        c.NuTauTT.n_hor_planes = 1
-        c.NuTauTT.n_vert_planes = 1 #simplified
+        c.NuTauTT.n_hor_planes = 4
+        c.NuTauTT.n_vert_planes = 4 #now same number, walls are squared
         c.NuTauTT.scifimat_width = 13.06 * u.cm # old value 13.045
```

Comment:
Why is this 13.06 instead of 13.045? Should we not use 13.06 in the calculations following?

Reply:
I used 13.06 in my calculations: 13.06 cm * 4 + 2.9 cm = 55.14 cm

Reply:
I do not know the origin of this change from 13.045 cm to 13.06
, but it is like that from 2019, according to git commit history: https://github.com/ShipSoft/FairShip/pull/227/files

Reply:
> I used 13.06 in my calculations: 13.06 cm * 4 + 2.9 cm = 55.14 cm

In that case it's even more important not to hardcode the values, but calculate them in the next lines, as the comments were wrong.